docs/topics/proc_deploying-application-to-standalone-rhel.adoc

```diff
@@ -0,0 +1,30 @@
+[id='deploying-application-to-standalone-rhel_{context}']
+= Deploying an application to standalone Red Hat Enterprise Linux
+
+You can deploy a {VertX} application to Red Hat Enterprise Linux.
+
+.Prerequisites
+
+* Red Hat Enterprise Linux 7
+* OpenJDK 8 installed
+* Maven 3.5 or later installed
+
+.Procedure
+
+. Download the {VertX} runtime from the link:https://maven.repository.redhat.com/[Red Hat Maven repository] to your RHEL 7 system.
+
+. Create a project for your application. Consider packaging your application as a fat jar to include all of the necessary dependencies.
```

Comment:
We're presuming the app exists or it was created according to [Creating a basic Vert.x application](https://launcher.fabric8.io/docs/vertx-runtime.html#creating-a-basic-vertx-application_vertx), so I think the first step isn't necessary.

I believe we should provide the command how to package your application. I think it should be `mvn clean package`, but i'm not 100% sure.